ci-operator/config/openshift/cluster-autoscaler-operator/openshift-cluster-autoscaler-operator-master.yaml

```diff
@@ -98,6 +98,81 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-nutanix
+- always_run: true
+  as: e2e-aws-periodic-pre
```

Comment:
For this repo, I would suggest having the AWS version as running always, but other platforms only on demand.

So for this one, just use `skip_if_only_changed` on AWS, drop `always_run`.

For the other platforms, drop the `skip_if_only_changed` and have `always_run: false`